### PR TITLE
Update fluentd-image.yml

### PR DESCRIPTION
**Pull request by MichaelBriggs-Coralogix**
Add workflow_dispatch option
Remove "latest" tag

### CODE REVIEW

File: .github/workflows/fluentd-image.yml [hunk 1]
```diff
@@ -40,6 +41,4 @@ jobs:
           context: ./logs/fluentd/image
           platforms: linux/amd64,linux/arm64
           push: true
-          tags: |
-            ${{ env.DOCKERHUB_REGISTRY }}:${{ steps.get-fluentd-version.outputs.version }}
-            ${{ env.DOCKERHUB_REGISTRY }}:latest
```
Comment (daidokoro) [inline .github/workflows/fluentd-image.yml:45]: Hey @MichaelBriggs-Coralogix ,

Do we want to remove the override for the latest tag? It what allow the user to get the most recent version when they pull image:latest.

Comment (MichaelBriggs-Coralogix) [reply .github/workflows/fluentd-image.yml:45]: Per my understanding, we're wanting to remove all "latest" tags from our Docker images (at least for our integrations) to ensure we explicitly set them in our charts. This is to ensure stability/supportability.

Comment (daidokoro) [reply .github/workflows/fluentd-image.yml:45]: Got it.

**Review by daidokoro — APPROVED** [review]
LGTM

**Pull request merged**